app/uk/gov/hmrc/cdsreimbursementclaim/models/eis/claim/Address.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaim.models.eis.claim
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class Address(
+  contactPerson: Option[String],
+  addressLine1: Option[String],
+  addressLine2: Option[String],
+  AddressLine3: Option[String],
```

Comment:
Should this be an uppercase A?

Reply:
actually, no - they designed the schema like that :man_shrugging: